doc/workshop/exercises/analysis/denoising/denoising.rst

```diff
@@ -108,9 +108,21 @@ add a few options to have finer control over what is written. These include:
    consume LAS.
 3. ``dataformat_id``: Format 0 supports both time and color information
 
+5. :ref:`writers.copc`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We will then turn the ``clean.laz`` file into a COPC file for vizualization with QGIS.
+
+::
```

Comment:
since this is json, we can actually do some syntax highlighting here, I think you can do

```rst
.. code-block:: json

     {
         "type": "writers.copc",
         "filename": "./exercises/analysis/colorization/clean.copc.laz"
     }
```